checkstyle-tester/README.md

```diff
@@ -62,15 +65,15 @@ Please use `-Dcheckstyle.consoleOutput=true` option if you need to see all Check
 groovy launch.groovy projects-to-test-on.properties my_check.xml -Dcheckstyle.consoleOutput=true
 ```
 
-**Attention:** this project by deafult uses released version of Checkstyle and sevntu.checkstyle
-If you you need to use custom (snapshot) versions please update pom.xml to reference that versions ([checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L29), [sevntu.checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L34)), and please make sure that custom(newly generated) versions are located in your local maven repo 
+**Attention:** this project by deafult uses released version of [Checkstyle](https://github.com/checkstyle/checkstyle/releases/latest) and [sevntu.checkstyle](https://github.com/sevntu-checkstyle/sevntu.checkstyle/releases/latest).
```

Comment:
`deafult` -> `default`

~~`uses released version` -> `uses latest released version`~~

> this project by deafult uses released version 

By default, Checkstyle use latest snapshot version, not latest release.
Only sevntu looks like it is based on latest release.